app/Http/Requests/PasswordResetLinkRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2025b\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PasswordResetLinkRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            "email" => "required|email|max:225",
```

Comment:
Please stick to one format, in other files I see arrayed rules, like `["required", "min:8", "max:225", "string"],`